### PR TITLE
update androidnativeplugin to 1.1.5

### LINKED ISSUE #2381: [ERROR] Sentry Example-App not building with latest Flutter Beta
### Platform

Flutter Mobile

### Obfuscation

Disabled

### Debug Info

Disabled

### Doctor

```
[✓] Flutter (Channel beta, 3.27.0-0.1.pre, on macOS 14.6.1 23G93 darwin-arm64, locale en-AT)
    • Flutter version 3.27.0-0.1.pre on channel beta at /Users/martin/fvm/versions/3.27.0-0.1.pre
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2e2c358c9b (8 days ago), 2024-10-22 11:02:13 -0400
    • Engine revision af0f0d559c
    • Dart version 3.6.0 (build 3.6.0-334.3.beta)
    • DevTools version 2.40.1

[!] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    • Android SDK at /Users/martin/Library/Android/sdk
    • Platform android-35, build-tools 35.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)
    ! Some Android licenses not accepted. To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 16.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 16B40
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2024.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)

[✓] VS Code (version 1.94.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.98.0

[✓] Connected device (3 available)
    • macOS (desktop)                 • macos                 • darwin-arm64   • macOS 14.6.1 23G93 darwin-arm64
    • Mac Designed for iPad (desktop) • mac-designed-for-ipad • darwin         • macOS 14.6.1 23G93 darwin-arm64
    • Chrome (web)                    • chrome                • web-javascript • Google Chrome 130.0.6723.70

[✓] Network resources
    • All expected network resources are available.

```

### Version

latest main branch

### Steps to Reproduce

Use the current Flutter Beta (3.27.0-0.1.pre) and try to build the sentry example app with `flutter build apk --debug` 

### Expected Result

Successful build.

### Actual Result

```
#: flutter build apk --debug
Changing current working directory to: /Users/martin/Repositories/sentry-dart/flutter/example

You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply

You are applying Flutter's main Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply

 android plugin version 8.3.2
:app:external cmake build 

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/martin/Repositories/sentry-dart/flutter/example/android/build.gradle' line: 29

* What went wrong:
A problem occurred evaluating root project 'android'.
> A problem occurred configuring project ':app'.
   > Could not resolve all dependencies for configuration ':app:releaseCompileClasspath'.
      > Could not find io.flutter:x86_release:1.0.0-af0f0d559c8a87d912a20971bbd84afc80a54b0f.
        Required by:
            project :app

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s
Running Gradle task 'assembleDebug'...                              3.8s
Gradle task assembleDebug failed with exit code 1
```

### Are you willing to submit a PR?

Yes

### PR DESCRIPTION
**Pull request by martinhaintz**
## :scroll: Description
updated the androidnativeplugin to 1.1.5

## :bulb: Motivation and Context
close #2381 

## :green_heart: How did you test it?
build it locally

## :pencil: Checklist
- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps

#skip-changelog

### CODE REVIEW

File: .github/workflows/flutter_test.yml
Comment (vaind) [inline .github/workflows/flutter_test.yml]: I believe this was an optimization for CI build times. Any reason to remove this?

Comment (martinhaintz) [reply .github/workflows/flutter_test.yml]: This was only added a few days ago by myself, because of the incompatibility of the latest flutter beta (`3.27.0-0.1.pre`) and the `androidNativeBundle`. The appending `targe-platform` was only a workaround to keep the ci pipeline working. 
As the maintainer of the plugin fixed this incompatibility, I will revert the changes.
Here is the [issue](https://github.com/getsentry/sentry-dart/issues/2381).

Comment (vaind) [reply .github/workflows/flutter_test.yml]: I see. On the other hand it **does** reduce build time by about 1.5 minute so I'd vote for keeping only the emulator build on. Your call

Comment (buenaflor) [reply .github/workflows/flutter_test.yml]: If it does reduce build time then I guess it's worth keeping

**Review by buenaflor — CHANGES_REQUESTED** [review]
as suggested let's keep `--target-platform=android-x64` as it is (even though unintended) optimization

**Review by buenaflor — APPROVED** [review]

**Pull request merged**
